tools/einsteinathome/pycbc_build_eah.sh

```diff
@@ -816,52 +825,30 @@ else
         fi
     fi
     echo -e ">> [`date`] git HEAD: `git log -1 --pretty=oneline --abbrev-commit`" >&3
-    sed -i~ s/func__fatal_error/func_fatal_error/ */gnuscripts/ltmain.sh
+    # fix typo in ltmain
+    # only helpful if something goes wrong, normally not needed
+    # sed -i~ s/func__fatal_error/func_fatal_error/ */gnuscripts/ltmain.sh
     if $build_dlls; then
-	git apply <<'EOF' || true
-From accb37091abbc8d8776edfb3484259f6059c4e25 Mon Sep 17 00:00:00 2001
```

Comment:
The patch would only be relevant for the Cygwin build, which is done basically with the latest LALSuite. Karl's patch was reverted in LALSuite on my request recently, as it caused problems on other platforms like OSX. The current version of the script relies on a newer commit:
https://versions.ligo.org/cgit/lalsuite/commit/?id=5a7ba7d6f3841b447d665b589f2ddf0561321e6c
but again, this is only relevant for the Cygwin build.
I'll at this (and another) commit to the script to be applied when it isn't (yet) there. Where do I get the "blessed" LALSuite version from? Is there a specific tag name?